.github/CONTRIBUTING.md

```diff
@@ -38,12 +41,88 @@ In case you were unable to determine which country the channel belongs to, you c
 
 - find out the full name of the channel and from which country it is broadcasting. This information can usually be found on [lyngsat.com](https://www.lyngsat.com/search.html) or [wikipedia.org](https://www.wikipedia.org/)
 - update the channel name if necessary
-- find the corresponding country code in [ISO_3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) table
+- find the corresponding ISO_3166-2 code for the country. You can find a full list of codes here: https://en.wikipedia.org/wiki/ISO_3166-2
 - open the `channels/` folder and find a file with the same name as the country code
 - at the very end of this file add a link to the channel with a description
 - commit all changes
 - send a pull request
 
+**Do you want to change the channel category?**
+
+- find the file that contains the channel. You can use a [GitHub Search](https://github.com/search/advanced?q=CHANNEL_NAME+repo%3Aiptv-org%2Fiptv+path%3A%2Fchannels&type=Code) to do this.
+- open the file
+- find the channel description
+- specify the appropriate category in the `group-title` attribute. A complete list of supported categories can be found [here](https://github.com/iptv-org/iptv#playlists-by-category).
+- commit all changes
+- send a pull request
+
+**Do you want to change the channel language?**
+
+- find the file that contains the channel. You can use a [GitHub Search](https://github.com/search/advanced?q=CHANNEL_NAME+repo%3Aiptv-org%2Fiptv+path%3A%2Fchannels&type=Code) to do this.
+- open the file
+- find the channel description
+- specify the appropriate language in the `tvg-language` attribute. The name of the language must comply with [ISO 639-3](https://iso639-3.sil.org/code_tables/639/data?title=&field_iso639_cd_st_mmbrshp_639_1_tid=94671&name_3=&field_iso639_element_scope_tid=All&field_iso639_language_type_tid=51&items_per_page=500) standart.
+- commit all changes
+- send a pull request
+
+If a channel is broadcast in several languages at once, you can specify them all through a semicolon, like this:
+
+```xml
+#EXTINF:-1 tvg-language="English;Chinese",CCTV
+http://example.com/cctv.m3u8
+```
+
+**Do you want to add a new EPG (Electronic Program Guide) source?**
+
+- check which country the EPG is intended for
```

Comment:
Weird formulation. "Check for which country the EPG is intented for."

Prefer using (maybe) : - check the channels listed in the EPG and check where they're broadcasted.